ts/features/payments/bizEventsTransaction/utils/index.ts

```diff
@@ -65,7 +65,7 @@ export const getPayerInfoLabel = (payer: InfoNotice["payer"]): string => {
   }
 
   const name = payer.name ? payer.name.trim() : "";
-  const taxCode = payer.taxCode ? `(${payer.taxCode.trim()})` : "";
+  const taxCode = payer.taxCode ? payer.taxCode.trim() : "";
```

Comment:
Since the tax code can coexist with the user's name, @thisisjp advised that if the user's name is present, the tax code should be displayed in parentheses (if provided). However, if only the tax code is supplied without the name, the parentheses should be omitted.